### ISSUE TITLE
[FVT]Please update case, failed to install xCAT 2.11 on ubuntu16.04 x86

### ISSUE THREAD

**Issue by caomengmeng** [xcat2/xcat-core#2310]
Description: Failed to install 2.11 xCAT with go-xcat using ``/go-xcat -x 2.11 -y install`` on ubuntu16.04-x86_64, because xCAT 2.11 not support Ubuntu Xenial.
Please update test case
```
RUN:xdsh c910f04x14v04 "cd /; ./go-xcat -x 2.11 -y install"
Running Time:5 sec
RETURN rc = 1
OUTPUT:
c910f04x14v04: Operating system:   linux
c910f04x14v04: Architecture:       x86_64
c910f04x14v04: Linux Distribution: ubuntu
c910f04x14v04: Version:            16.04
c910f04x14v04:
c910f04x14v04: Reading repositories ...... failed
c910f04x14v04: Get:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]
c910f04x14v04: Ign:2 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial InRelease
c910f04x14v04: Hit:3 http://archive.ubuntu.com/ubuntu xenial InRelease
c910f04x14v04: Get:4 http://xcat.org/files/xcat/repos/apt/xcat-dep xenial InRelease [1921 B]
c910f04x14v04: Hit:5 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
c910f04x14v04: Ign:6 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial Release
c910f04x14v04: Ign:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Get:10 http://xcat.org/files/xcat/repos/apt/xcat-dep xenial/main amd64 Packages [3159 B]
c910f04x14v04: Ign:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Ign:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Ign:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Ign:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Err:7 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main amd64 Packages
c910f04x14v04:   404  Not Found
c910f04x14v04: Ign:8 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main all Packages
c910f04x14v04: Ign:9 http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial/main Translation-en
c910f04x14v04: Fetched 107 kB in 1s (77.4 kB/s)
c910f04x14v04: Reading package lists...
c910f04x14v04: W: The repository 'http://xcat.org/files/xcat/repos/apt/2.11/xcat-core xenial Release' does not have a Release file.
c910f04x14v04: W: http://xcat.org/files/xcat/repos/apt/xcat-dep/dists/xenial/InRelease: Signature by key F75B1BF678B644FDF3AACFC860A3E9ACC6565BC9 uses weak digest algorithm (SHA1)
c910f04x14v04: E: Failed to fetch http://xcat.org/files/xcat/repos/apt/2.11/xcat-core/dists/xenial/main/binary-amd64/Packages  404  Not Found
c910f04x14v04: E: Some index files failed to download. They have been ignored, or old ones used instead.
CHECK:rc == 0   [Failed] 
```

**Comment by tingtli** [comment]
Fixed on #2332

**Comment by caomengmeng** [comment]
Closed
